moped-database/migrations/default/1750186599182_set_should_sync_ecapris_false/down.sql

```diff
@@ -0,0 +1 @@
+alter table "public"."moped_project" alter column "should_sync_ecapris_statuses" set default 'true';
```

Comment:
If desired we could suppress  triggers activity logs using the old `SET session_replication_role = replica;` trick

Reply:
i say let's go for it! i often forget about this trick so this will help me keep it in mind. 🙏

Reply:
done!